wagn/rails/controllers/card_controller.rb

```diff
@@ -248,7 +243,9 @@ def show view = nil, status = 200
     else
       args = { :text=>result, :status=>status }
       args[:content_type] = 'text/text' if format == :file
-      render args
+      card.run_callbacks :render_view do
```

Comment:
"read" doesn't work because then API would be `:before=>:read, :on=>:read`. I chose "render_view" because it's not before/after "show" it's before/after we render the view in show.